app/src/main/java/io/zenandroid/onlinego/ui/screens/settings/CustomAdapter.kt

```diff
@@ -0,0 +1,66 @@
+package io.zenandroid.onlinego.ui.screens.settings
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.Build
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.ImageView
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import io.zenandroid.onlinego.R
+import io.zenandroid.onlinego.data.model.BoardTheme
+
+class CustomAdapter(bordThemes: Array<BoardTheme>, mContext: Context) :
```

Comment:
Sometimes I forget how impossibly complicated the legacy view system is... At some point I'll have to reimplement the settings in Compose as displaying a list should not be that complicated...

Reply:
And I didn't even manage to put a grid layout on this dialog.... It is hell for me 🔥